src/query/service/src/pipelines/builders/copy.rs

```diff
@@ -205,23 +252,14 @@ pub fn build_append_data_with_finish_pipeline(
                         // 2. Try to purge copied files if purge option is true, if error will skip.
                         // If a file is already copied(status with AlreadyCopied) we will try to purge them.
                         if purge {
-                            try_purge_files(ctx.clone(), &stage_info_clone, &files).await;
-                        }
-
-                        // Status.
-                        {
-                            info!("all copy finished, elapsed:{}", start.elapsed().as_secs());
```

Comment:
I think we need debug info here.

Reply:
I will add some metrics for COPY in another PR, it will introduce more information and metrics to measure COPY(including distributed).